src/lib/genesis_epoch_data/genesis_epoch_data.ml

```diff
@@ -0,0 +1,11 @@
+module Data = struct
```

Comment:
It's a little weird that this is it's own library. It might be better placed in `Coda_state`.

Reply:
Will move it to `consensus` since it is consensus related and also since there's a dependency cycle between `coda_state` and `consensus`